packages/@lwc/template-compiler/src/__tests__/parser.spec.ts

```diff
@@ -548,10 +548,12 @@ describe('props and attributes', () => {
         expect(root.children[0].properties).toMatchObject([
             {
                 name: 'ariaHidden',
+                attributeName: 'aria-hidden',
```

Comment:
I know we're moving away from the parser spec but I wanted to add this as a reminder to check for it once we move to the fixtures based testing.